src/test/CMakeLists.txt

```diff
@@ -530,15 +530,18 @@ if(WITH_RBD)
   add_ceph_test(run-rbd-unit-tests.sh ${CMAKE_CURRENT_SOURCE_DIR}/run-rbd-unit-tests.sh)
   if(FREEBSD)
     add_ceph_test(rbd-ggate.sh ${CMAKE_CURRENT_SOURCE_DIR}/rbd-ggate.sh)
+    add_test_label(rbd-ggate.sh "vstart;freebsd")
```

Comment:
the "freebsd" makes sense to me. as it is able to single out the test not exercised by non-freebsd builder.

but why do we need "vstart" label? how is it supposed to be used?

Reply:
@tchaikov
I'm using the vstart label to only (re)run tests that build a mini-cluster. No use in running them if msgr-code does not work. The problem I have on and off with the msgr2 code.

Reply:
so your intention is to filter out the tests which uses msgr when you are fixing mgr2  support. i think the right way to address this is not to upstream this change, but to fix msgr2.

Reply:
@tchaikov 
As an example, which I will not submit as PR.
Some of the tests always fail when running in `ctest -j xx`. I have not yet really looked into it.
But marking those tests `serial` allows me to exclude the tests from the parallel run, and do them one at the time.
Same for vstart. I do not always want all the tests which spawn a cluster.
This allow you to exclude it.
Very handy during running debugging tests run.